### PR TITLE
CONFIG: RAIL-1440, RAIL-1408, RAIL-1357, BB-1432, BB-1428 Update of translations

### PR DESCRIPTION
**Pull request by sitina**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3772/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3022/ : SUCCESS in 7m 13s

File: src/translations/fr-FR.json [hunk 1]
```diff
@@ -54,12 +54,13 @@
...
-   "visualizations.measure.arithmetic.change.title": "Variation de {secondMeasureTitle} à {firstMeasureTitle}",
-   "visualizations.legend.text.top": "Haut",
-   "visualizations.legend.text.bottom": "Bas",
-   "visualizations.legend.text.left": "Gauche",
-   "visualizations.legend.text.right": "Droite",
-   "visualizations.legend.text.colon": " :",
-   "visualizations.legend.text.combo": "{0} ({1})",
-   "visualizations.menu.aggregations": "Agrégations"
+   "visualizations.measure.arithmetic.change.title": "Passer de {secondMeasureTitle} à {firstMeasureTitle}",
+   "visualizations.legend.text.top": "Haut :",
```
Comment (petrjanu) [inline src/translations/fr-FR.json]: extra space before `:`

File: src/translations/fr-FR.json [hunk 2]
```diff
@@ -54,12 +54,13 @@
...
    "visualizations.measure.arithmetic.ratio.title": "Ratio de {firstMeasureTitle} et {secondMeasureTitle}",
-   "visualizations.measure.arithmetic.change.title": "Variation de {secondMeasureTitle} à {firstMeasureTitle}",
-   "visualizations.legend.text.top": "Haut",
-   "visualizations.legend.text.bottom": "Bas",
-   "visualizations.legend.text.left": "Gauche",
-   "visualizations.legend.text.right": "Droite",
-   "visualizations.legend.text.colon": " :",
-   "visualizations.legend.text.combo": "{0} ({1})",
-   "visualizations.menu.aggregations": "Agrégations"
+   "visualizations.measure.arithmetic.change.title": "Passer de {secondMeasureTitle} à {firstMeasureTitle}",
```
Comment (petrjanu) [inline src/translations/fr-FR.json]: extra space before `:`

File: src/translations/fr-FR.json [hunk 3]
```diff
@@ -54,12 +54,13 @@
...
+   "visualizations.legend.text.top": "Haut :",
+   "visualizations.legend.text.bottom": "Bas :",
```
Comment (petrjanu) [inline src/translations/fr-FR.json]: extra space before `:`

File: src/translations/fr-FR.json [hunk 4]
```diff
@@ -54,12 +54,13 @@
...
+   "visualizations.legend.text.left": "Gauche :",
```
Comment (petrjanu) [inline src/translations/fr-FR.json]: extra space before `:`

File: src/translations/fr-FR.json [hunk 5]
```diff
@@ -54,12 +54,13 @@
...
+   "visualizations.legend.text.right": "Droite :",
```
Comment (petrjanu) [inline src/translations/fr-FR.json]: extra space before `:`

**Comment by sitina** [comment]
@petrjanu updated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3783/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3035/ : SUCCESS in 10m 23s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/825/ : SUCCESS in 1m 01s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3038/ : SUCCESS in 15m 56s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1618/ : SUCCESS in 37m 36s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/826/ : SUCCESS in 36m 58s

**Pull request merged**